### PR TITLE
4.0] Active indicator on wrong side in RTL

### PR DESCRIPTION
**Pull request by brianteeman**
Pull Request for Issue #24704

### Summary of Changes
See #24704 for details

### Testing Instructions

npm i

### CODE REVIEW

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 8a6f4b2e25d679a9e79b1adc1e1a3321558f8949<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27405">issues.joomla.org/tracker/joomla-cms/27405</a>.</sub>

**Comment by brianteeman** [comment]
@wilsonge its still failing but with the usual timeout issue - its not getting as far as the relevant test

**Comment by infograf768** [comment]
Please post screenshot. What is the `active indicator` ?

**Comment by Quy** [comment]
See #27404

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 8a6f4b2e25d679a9e79b1adc1e1a3321558f8949<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27405">issues.joomla.org/tracker/joomla-cms/27405</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27405">issues.joomla.org/tracker/joomla-cms/27405</a>.</sub>

File: administrator/templates/atum/scss/blocks/_sidebar.scss [hunk 1]
```diff
@@ -273,10 +273,15 @@
           position: absolute;
           top: 0.15rem;
           bottom: 0.15rem;
-          left: 0;
           width: 0.25rem;
           content: "";
           background-color: var(--atum-contrast);
+          [dir=ltr] & {
```
Comment (infograf768) [inline administrator/templates/atum/scss/blocks/_sidebar.scss:280]: Please add empty line before `[dir=ltr] & {`

File: administrator/templates/atum/scss/blocks/_sidebar.scss [hunk 2]
```diff
@@ -273,10 +273,15 @@
...
+            left: 0;
+          }
+          [dir=rtl] & {
```
Comment (infograf768) [inline administrator/templates/atum/scss/blocks/_sidebar.scss:284]: Please add empty line before `[dir=rtl] & {`

**Comment by infograf768** [comment]
OK to go after cs is corrected.

**Comment by infograf768** [comment]
@wilsonge @HLeithner 
Good to go. (Drone failing unrelated)

**Comment by HLeithner** [comment]
Thanks

**Pull request merged**